templates/openwisp2/settings.py

```diff
@@ -128,6 +128,8 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
+SPATIALITE_LIBRARY_PATH = 'mod_spatialite'
```

Comment:
Should be:

`SPATIALITE_LIBRARY_PATH = '{{ openwisp2_spatialite_path }}'`

Do you understand why?

Reply:
Ah, so it takes the value from the yml settings file instead of hardcoding it? I see...